### DISCUSSION TITLE
Host key verification failed on jenkins

### DISCUSSION THREAD

**Discussion by hassan-munir-confiz** [community/community#51216] [Other Feature Feedback, Questions, & Ideas]
### Updation Github RSA SSH host key

As per latest news due to recent RSA SSH host key updatation we have been facing a terrifying error. 

We have fixed the error by updating ssh key on know_hosts file as per below blog instruction:

https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/

But still getting the error "Host Key verification failed". 

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/46344843/228217603-3020a0eb-5205-4fd4-a26d-453e9bbf8feb.png)

Please let me know if i miss anything.

Thanks

**Comment by airtower-luna** [comment]
Did you update the `known_hosts` file used by the Jenkins user as well?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hassan-munir-confiz ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
